source/patterns/@aws-solutions-constructs/core/lib/lambda-event-source-mapping-defaults.ts

```diff
@@ -13,16 +13,9 @@
 
 import * as lambda from '@aws-cdk/aws-lambda';
 import { overrideProps } from './utils';
-import { DynamoEventSourceProps, S3EventSourceProps } from '@aws-cdk/aws-lambda-event-sources';
+import { DynamoEventSourceProps, S3EventSourceProps, KinesisEventSourceProps } from '@aws-cdk/aws-lambda-event-sources';
 import * as s3 from '@aws-cdk/aws-s3';
 
-export function DefaultKinesisEventSourceProps(_eventSourceArn: string) {
```

Comment:
This function was only being used by the pattern (which now uses `KinesisEventSourceProps`), so I deleted it.